pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -254,7 +254,7 @@ func calculateIPOffset(base *big.Int, ip net.IP) int {
 func RangeSize(subnet *net.IPNet) int64 {
 	ones, bits := subnet.Mask.Size()
 	if (bits - ones) >= 31 {
-		panic("masks greater than 31 bits are not supported")
+		panic("masks shorter than /2 for IPv4 or /98 for IPv6 are not supported.")
```

Comment:
Change looks fine. Is there a meaningful name that could be used for the "31" limit to make this a bit more understandable? MaxHosts? MaxHostBits?

Should this function return an error and be handled by caller, versus using panic? Or are there too many callers for it to change?